png/apng/timeout.txt

```diff
@@ -0,0 +1,31 @@
+SET TIMEOUT: png/apng/acTL-plays-one.html
```

Comment:
This file seems redundant?

Reply:
Why? It is a test for
    Number of frames: 2
    Number of plays: 1
so, two frames, plays the sequence once. A non-APNG implementation would display the static image only, and thus fail this test.

Reply:
I meant the `png/apng/timeout.txt` file.

Reply:
Oh right. That was a temp file to create entries for the whitelist. It shouldn't be part of this PR. I will delete it.

Reply:
deleted